CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,12 @@
+# Community Participation Guidelines
+
+This repository is governed by Mozilla's code of conduct and etiquette guidelines.
+For more details, please read the
+[Mozilla Community Participation
+Guidelines](https://www.mozilla.org/about/governance/policies/participation/).
+
+## How to Report
+For more information on how to report violations of the Community Participation Guidelines, please
+read our [How to Report](https://www.mozilla.org/about/governance/policies/participation/reporting/)
```

Comment:
I've checked other repo, and they have single quote around How to Report link

https://github.com/mozilla-services/kinto-amo/blob/master/CODE_OF_CONDUCT.md
https://github.com/mozilla/multi-account-containers/blob/master/CODE_OF_CONDUCT.md
https://github.com/mozilla/diversity/blob/master/CODE_OF_CONDUCT.md

also, to follow them, might be better just removing all line-wrap inside both paragraph.